packages/commonwealth/server/passport/httpBearer.ts

```diff
@@ -0,0 +1,15 @@
+import passport from 'passport';
+import { Strategy as BearerStrategy } from 'passport-http-bearer';
+
+// bearer return 401 if token is invalid
+export function initBearerAuth(secretName, secretValue) {
+  passport.use(
+    secretName,
+    new BearerStrategy((token, done) => {
```

Comment:
@NakulManchanda can we add a check for undefined secret values? To forbid usage of the route in the case where we forget to set the env var.

Reply:
when undefined it fails with 400 like it is right now

Reply:
let me try adding that check 
- to not register strategy when secretName or secretValue, 
- and check what happens on using authentication strategy which is not registered

Reply:
- check was already covered in `BearerStrategy` - 401 or 400 for undefined
- still added an extra check(s)